CMSgov/onemac
# Oy2 916 1033 rai emails

PR test link: https://dstisstnuy7o5.cloudfront.net

To test emails sent to CMS, update the CircleCI environment variable: OY2_916_1033_RAI_emails_CMS_SPA_FORM_CMS_EMAIL  to your email address and rebuild.

OY2 - 916
- Demonstrate that an SPA RAI submission confirmation email is sent to the submitting user
- Demonstrate that an SPA RAI submission details are sent to the CMS inbox

OY2 - 1033
- Demonstrate that a Waiver RAI submission confirmation email is sent to the submitting user
- Demonstrate that Waiver RAI submission details are sent to the CMS inbox

Changes;
- Added WaiverRAI changeRequest Type to backend (SPARAI was there from last PR)
- Removed urgency functions from SPA email template (got removed from form)
- Updated email content to improve visual grouping of labels and text.
- Added WaiverRAI email template
- Updated SPARAI email template (was not made specific to SPA RAI before)
- Updated getEmailTemplates to include the new functions.

Test:

1. I used two email addresses, one I set to be “CMS” and the other the “State”  -- can use one email, just have to keep things straight :)
2. To test emails sent to CMS, update the CircleCI environment variable: OY2_916_1033_RAI_emails_CMS_SPA_FORM_CMS_EMAIL  to your email address and rebuild.
3. Login to application.
4. Create a SPA RAI and check both emails for content and clarity
5. Create a Waiver RAI and check both emails for content and clarity

PR test link: https://dstisstnuy7o5.cloudfront.net
